llvm/lib/Transforms/Vectorize/VPRecipeBuilder.h

```diff
@@ -34,6 +53,9 @@ class VPRecipeBuilder {
   /// Target Library Info.
   const TargetLibraryInfo *TLI;
 
+  // Target Transform Info
+  const TargetTransformInfo *TTI;
```

Comment:
Not used in the current version, but could if `getScaledReduction` is moved to builder?

Reply:
Yeah it is indeed needed after moving `getScaledReduction`.